pages/19.Translations/docs.en.md

```diff
@@ -60,4 +60,9 @@ A language is downloaded automatically every time the configuration is saved and
 
 The language should be downloaded again with the latest translations. The translations are generated from Transifex once a day.
 
-If you have any questions about translations, join the community in the [Slack #Translations channel](https://www.mautic.org/slack/).
+If you have any questions about translations, join the community in the [Slack #Translations channel][Slack #Translations channel].
+
+
+[Transifex]:<https://www.transifex.com/mautic/mautic/>
+[Transifex Documentation]:<http://docs.transifex.com/tutorials/txeditor/>
+[Slack #Translations channel]:<https://www.mautic.org/slack/>
```

Comment:
Can we change this to a direct link to the channel: https://mautic.slack.com/archives/C02HV79J2